### PR TITLE
Added event cases for feature results and georef results

### PR DESCRIPTION
**Pull request by Sorrento110**
Also process mode now adds a sample feature to the CDR.

### CODE REVIEW

File: sample_cdr_georef/sample_cdr_georef/server.py [hunk 1]
```diff
@@ -237,6 +237,92 @@ async def georeference_map(req: MapEventPayload,  response_model=GeoreferenceRes
     return results
 
 
+def dummy_feature_results():
```
Comment (jgawrilo) [inline sample_cdr_georef/sample_cdr_georef/server.py]: Does this result 'make sense' for someone looking at it? I.e., this dummy result doesn't supply map_unit information so '11' doesn't mean anything.

File: sample_cdr_georef/sample_cdr_georef/server.py [hunk 2]
```diff
@@ -237,6 +237,92 @@ async def georeference_map(req: MapEventPayload,  response_model=GeoreferenceRes
...
+                    },
+                    "properties": {}
+                }
+                ]
+            },
+        ]
+    }
+
+    feature_payload = FeatureResults(
+        cog_id="6657_9294",
```
Comment (jgawrilo) [inline sample_cdr_georef/sample_cdr_georef/server.py]: @Sorrento110 This should be read programmatically from the event payload - not hardcoded.  I.e., if this event is for another cog_id, the submitted result is wrong.  Also this format looks like the ngmdb_prodid_ngmdb_item_id - which is NOT the cog_id.

File: sample_cdr_georef/sample_cdr_georef/server.py [hunk 3]
```diff
@@ -237,6 +237,92 @@ async def georeference_map(req: MapEventPayload,  response_model=GeoreferenceRes
...
+        polygon_feature_results=[polygon_feature_1, polygon_feature_2],
+        system=app_settings.system_name,
+        system_version=app_settings.system_version
+    )
+
+    return feature_payload
+
+
+async def post_feature_results():
```
Comment (jgawrilo) [inline sample_cdr_georef/sample_cdr_georef/server.py]: @Sorrento110 can we at least download the cog to show what a client would _actually_ do similar to https://github.com/DARPA-CRITICALMAAS/cdr_client_examples/blob/main/sample_cdr_georef/sample_cdr_georef/server.py#L201

File: sample_cdr_georef/sample_cdr_georef/server.py [hunk 4]
```diff
@@ -325,3 +421,4 @@ def register_system():
     if args.mode == 'process':
```
Comment (jgawrilo) [inline sample_cdr_georef/sample_cdr_georef/server.py:470]: @Sorrento110 think you could add the 'getter of results' calls to the 'process' path as well please?

File: sample_cdr_georef/sample_cdr_georef/server.py (hunk 1 above)
Comment (Sorrento110) [reply sample_cdr_georef/sample_cdr_georef/server.py]: I have now replaced this with the object that is expected in the FeatureResults schema.

**Pull request merged**